File: packages/synapse-sdk-ts/examples/quickstart.ts
import { init } from "../src/index.js";

async function main() {
  const synapse = init({
    apiUrl: "http://localhost:8080",
    projectId: "water_delivery_logistics"
  });

  synapse.capture({
    event_type: "agent_message",
    payload: { text: "Omega gate now requires access cards" },
    observed_at: new Date().toISOString(),
    tags: ["demo", "omega"]
  });

  await synapse.flush();
  console.log("Flushed one event to Synapse API");
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});

File: packages/synapse-sdk-py/examples/quickstart.py
from __future__ import annotations

from synapse_sdk import SynapseConfig, init

synapse = init(
    SynapseConfig(
        api_url="http://localhost:8080",
        project_id="water_delivery_logistics",
    )
)

synapse.capture(
    event_type="agent_message",
    payload={"text": "Omega gate now requires access cards"},
    tags=["demo", "omega"],
)

synapse.flush()
print("Flushed one event to Synapse API")
